finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/HyperLogLog.scala

```diff
@@ -0,0 +1,49 @@
+package com.twitter.finagle.redis.protocol
+
+import com.twitter.finagle.redis.ClientError
+import com.twitter.finagle.redis.protocol._
+import org.jboss.netty.buffer.{ChannelBuffers, ChannelBuffer}
+
+case class PFAdd(key: ChannelBuffer, elements: Seq[ChannelBuffer]) extends StrictKeyCommand {
+  val command = Commands.PFADD
+
+  override def toChannelBuffer = RedisCodec.toUnifiedFormat(Seq(CommandBytes.PFADD, key) ++ elements)
+}
+
+object PFAdd {
+  def apply(args: Seq[Array[Byte]]): PFAdd = args match {
+    case head :: tail =>
+      PFAdd(ChannelBuffers.wrappedBuffer(head), tail map ChannelBuffers.wrappedBuffer)
+
+    case _ =>
+      throw ClientError("Invalid use of PFAdd")
```

Comment:
scaladoc @throws?

(side: is this how we do things in finagle-redis, otherwise I'd prefer Try/Option)